app/views/application/_input.html.erb

```diff
@@ -0,0 +1,3 @@
+<% name = local_assigns.fetch(:name) %>
+<% value = local_assigns.fetch(:value) %>
+<input type="text" class="form-control" name="<%= name %>" value="<%= value %>" />
```

Comment:
we can use `text_field_tag` instead of this partial: https://apidock.com/rails/ActionView/Helpers/FormTagHelper/text_field_tag